pkg/api/vlabs/validate.go

```diff
@@ -458,16 +458,6 @@ func (a *Properties) validateAgentPoolProfiles(isUpdate bool) error {
 			return e
 		}
 
-		if to.Bool(agentPoolProfile.AcceleratedNetworkingEnabled) || to.Bool(agentPoolProfile.AcceleratedNetworkingEnabledWindows) {
-			if a.IsAzureStackCloud() {
-				return errors.Errorf("AcceleratedNetworkingEnabled or AcceleratedNetworkingEnabledWindows shouldn't be set to true as feature is not yet supported on Azure Stack")
```

Comment:
I think we still need to error is it is Azure stack.  @jadarsie can confirm

Reply:
thx @jsturtevant, that's correct, please keep AzureStack validations

Reply:
reverted it